ImperatorToCk3Tests/ImperatorWorldTests/Families/FamilyTests.cpp

```diff
@@ -0,0 +1,115 @@
+#include "gtest/gtest.h"
+#include "../ImperatorToCk3/Source/Imperator/Families/Family.h"
+#include <sstream>
+
+TEST(ImperatorWorld_FamilyTests, IDCanBeSet)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Family theFamily(input, 42);
+
+	ASSERT_EQ(theFamily.getID(), 42);
```

Comment:
Best to reverse the conditions in the ASSERT here. When a test fails, the results assume the first item is the expected output and the second is the actual. It's easier to debug if you're in the habit of writing it that way.

Ditto for all other tests using ASSERT_EQ.